pegen/pegen.c

```diff
@@ -304,6 +304,49 @@ endmarker_token(Parser *p)
     return expect_token(p, ENDMARKER);
 }
 
+static const char *reserved_keywords[] = {
+    "and",
```

Comment:
Idea: Maybe we should have a way to automatically generate this list from the grammar. Maybe we could make this list available from the generated code...

Reply:
Honestly I'm not sure if that's worth the effort. Sure, `pgen` generates the list of keywords automatically from the grammar. But that harks back to a time when adding a new keyword was no big deal. The last time we added new keywords it was a big mess (`async`/`await`) and those still aren't proper keywords (due to the `feature_version` flag on `ast.parse()`).